### PR TITLE
Upload error pages to S3

### PR DESCRIPTION
**Pull request by bilbof**
This will upload our error pages e.g. public/404.html to S3, so CloudFront can serve them, rather than the app.

This follows alphagov/govuk-coronavirus-business-volunteer-form#327 and alphagov/covid-engineering#229

https://trello.com/c/di3xSyYN/385-serve-error-pages-from-s3

### CODE REVIEW

File: concourse/pipeline.yml [hunk 1]
```diff
@@ -118,6 +118,7 @@ jobs:
           SECRET_KEY_BASE: ((secret-key-base-staging))
           AWS_DYNAMODB_SUBMISSIONS_TABLE_NAME: coronavirus-vulnerable-people-staging
           AWS_ASSETS_BUCKET_NAME: govuk-coronavirus-vulnerable-people-assets-staging
+          AWS_ERROR_PAGES_BUCKET_NAME: govuk-coronavirus-business-volunteer-error-pages-staging
```
Comment (Rosa-Fox) [inline concourse/pipeline.yml]: should this be `govuk-coronavirus-vulnerable-people-error-pages-staging`?

File: concourse/pipeline.yml [hunk 2]
```diff
@@ -182,6 +183,7 @@ jobs:
           SECRET_KEY_BASE: ((secret-key-base-prod))
           AWS_DYNAMODB_SUBMISSIONS_TABLE_NAME: coronavirus-vulnerable-people-prod
           AWS_ASSETS_BUCKET_NAME: govuk-coronavirus-vulnerable-people-assets-prod
+          AWS_ERROR_PAGES_BUCKET_NAME: govuk-coronavirus-business-volunteer-error-pages-prod
```
Comment (Rosa-Fox) [inline concourse/pipeline.yml]: should this be govuk-coronavirus-vulnerable-people-error-pages-prod?

File: concourse/tasks/deploy-to-govuk-paas.yml [hunk 3]
```diff
@@ -65,6 +67,8 @@ run:
       cf set-env govuk-coronavirus-vulnerable-people-form GOVUK_NOTIFY_SMS_TEMPLATE_ID "$GOVUK_NOTIFY_SMS_TEMPLATE_ID"
       cf set-env govuk-coronavirus-vulnerable-people-form NOTIFY_API_KEY "$NOTIFY_API_KEY"
       cf set-env govuk-coronavirus-vulnerable-people-form PAAS_ENV "$CF_SPACE"
+      cf set-env govuk-coronavirus-business-volunteer-form AWS_ERROR_PAGES_BUCKET_NAME "$AWS_ERROR_PAGES_BUCKET_NAME"
+      cf set-env govuk-coronavirus-business-volunteer-form AWS_REGION "$AWS_REGION"
```
Comment (Rosa-Fox) [inline concourse/tasks/deploy-to-govuk-paas.yml]: `cf set-env govuk-coronavirus-vulnerable-people-form `?

**Comment by bilbof** [comment]
@Rosa-Fox thanks! Good spot, I've fixed those 😃

**Review by Rosa-Fox — APPROVED** [review]

**Pull request merged**